samples/bluetooth/direction_finding_connectionless_tx/README.rst

```diff
@@ -38,6 +38,15 @@ Configuration
 
 |config|
 
+This sample configuration is split into the following two files:
+
+* generic configuration is available in :file:`prj.conf` file
+* board specific configuration is avaialbe in :file:`boards/<BOARD>.conf` file
+
+Board specific configuration involves configuring the Bluetooth LE controller configuration.
```

Comment:
Something must have went wrong here... Seems to be one configuration too much, at the end of the sentence.

Reply:
I'm sorry. It should be correct now.